### DISCUSSION TITLE
Change Date formats in Query results

### DISCUSSION THREAD

**Discussion by KevinWellsMLC** [dbeaver/dbeaver#10691] [Q&A] [answered]
In the query results , currently date fields are showing in 12-14-2020 10:03:42 format.   I’d like them to display in 14-DEC-2020 10:21:07 AM format.   I’ve attempted to change the format in the format/view –> data format –> both global and datasource settings but have had no success.

Is it possible to change?

System information:   	
    Windows 2012 R2     	
    DBeaver 7.2.4

Connection specification:     	
    Database Oracle 11g

Thanks you.

**Comment by kseniiaguzeeva** [comment]
Most probably you change the format via settings button in the bottom toolbar. It doesn't work for now https://github.com/dbeaver/dbeaver/issues/10601

Please right-click the connection name in DB Navigator>select Edit connection>set you date/time format>Press Test connection and press Ok after it.

The changes will be applied.

Please feel free to ask to reopen the ticket if the issue is not resolved.

**Comment by KevinWellsMLC** [comment]
I followed your instructions.   The date format is still not showing correctly in my query results.

**Comment by kseniiaguzeeva** [comment, marked as answer]
1. Please check also that this option is disabled. 
2. How do you write timestamp pattern? Here is my settings

![Screenshot 2020-12-22 at 09 58 45](https://user-images.githubusercontent.com/51405061/102859743-8e897900-443d-11eb-9903-a443b2ba11c3.png)

![Screenshot 2020-12-22 at 10 01 01](https://user-images.githubusercontent.com/51405061/102859516-128f3100-443d-11eb-929e-c17d721cac9d.png)

**Comment by KevinWellsMLC** [comment]
This fixed my issues.

Thank you.

**Comment by kseniiaguzeeva** [comment]
Great!
